projects/LandMine.md

```diff
@@ -0,0 +1,92 @@
+## Project Name <!-- Add your project name here with format "Project Name"-->
+
+LandMine
+
+## Category
+<!--developer tooling, application, wallet, infrastructure, etc-->
+Application
+
+## Project Description
+<!--Describe your project in a few sentences. -->
+Landmine is a platform where each property {like apartments,bunglow} is represented as an nft and then these nft are put up for sale.
+-Buyers can buy the properties listed on the site.
+-After Buyer agress to buy the property they will upload the agreement details which will be saved in the ipfs platform and seller will complete the transaction after the completion of payment.
+-Buyer can verify these transaction details by using their property id by entering their id and they will get the ipfs link of their agreement.
+
+### Use of IPFS
+- We have used ipfs for storing the documents related to the agreement details .
```

Comment:
Can you share a link to this? I did not see in the Github repo or links in this document.

Reply:
l have updated it now. here is the <a href="https://gateway.ipfs.io/ipfs/QmTiKQjNEAeNpxdkMNvWpKj4jbz8cRaEp4EjQAyAHYURAT">link</a>